templates/deployment.yaml

```diff
@@ -49,6 +49,10 @@ spec:
               port: 8080
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+            - name: branddesc
+              mountPath: "/usr/share/nginx/static/frontend/brand_desc.html"
+              subPath: brand_desc
```

Comment:
does this work? you have the mountPath set to a filename and the subPath set as well. I don't think you need both. I think what this is doing right now is using `brand_desc.html` as a directory lol

Reply:
Yea, verify this.  I if you mount a configmap, and specify a directory, each key-value pair becomes a file.  You can have the key named `brand_desc.html`. The subpath is not necessary.

Reply:
Good catch. It worked, but maybe because I don't need subPath.